src/verifications/verifications.ts

```diff
@@ -2061,15 +2149,16 @@ export class Verifications extends Logger {
     disabelSubVerifications = false,
```

Comment:
maybe we can change the "disabelSubVerifications" to the correct phrase "disableSubVerifications"